### DISCUSSION TITLE
--hostname option works on macOS but uses localhost on win 10 and WSL. What might be the cause?

### DISCUSSION THREAD

**Discussion by borsemayur2** [alephjs/aleph.js#193] [Q&A] [answered]
My app always run on http://localhost:8080 even after setting -hn option with some value. Due to this, I'm not able to run the app in docker.

**Comment by jupegarnica** [comment, marked as answer]
I PR this.  
https://github.com/alephjs/aleph.js/pull/167
should work in next release

**Reply by borsemayur2** [reply]
> 
> 
> I PR this.
> #167
> should work in next release

It's working now. Thanks
